BudgetTracker.Common/Exceptions/ValidationException.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace BudgetTracker.Common.Exceptions
+{
+    public class ValidationException : Exception
+    {
+        public ValidationException(string message)
```

Comment:
Do we want to have some logging here? / Shouldn't we consider setting up some sort of logging (If we want to use this for future long term use) 

Don't worry about changing it here might create a card to set up logging.

Reply:
Yeah I agree. I just created them:

https://github.com/BudgetSquirrel/BudgetTracker/issues/93
https://github.com/BudgetSquirrel/ApplicationWebApi/issues/13